explorer/src/lib/components/provisioners-card/ProvisionersCard.svelte

```diff
@@ -0,0 +1,69 @@
+<svelte:options immutable={true} />
+
+<script>
+  import {
+    DataCard,
+    ProvisionersList,
+    ProvisionersTable,
+  } from "$lib/components";
+
+  import "./ProvisionersCard.css";
+
+  /** @type {HostProvisioner[] | null}*/
+  export let provisioners;
+
+  /** @type {Error | null}*/
+  export let error;
+
+  /** @type {Boolean} */
+  export let loading;
+
+  /** @type {boolean} */
+  export let isSmallScreen;
+
+  const ITEMS_TO_DISPLAY = 15;
```

Comment:
Seems this is the same for all provisioners, transactions and blocks. I would move it to a constants file and then reuse it in all of the components.